### PR TITLE
Basic support for Sophos xg v18

### PR DESCRIPTION
**Pull request by corsoblaster**
Please give a short description what your pull request is for

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)
- [x] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: includes/definitions/sophos-xg.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+icon: sophos-xg
+group: sophos
+over:
+    - { graph: device_bits, text: 'Device Traffic' }
+    - { graph: device_processor, text: 'CPU Usage' }
+    - { graph: device_mempool, text: 'Memory Usage' }
+discovery:
+    - sysObjectID:
+        - .1.3.6.1.4.1.2604.5
+mib_dir: sophos-xg
```
Comment (Jellyfrog) [inline includes/definitions/sophos-xg.yaml:13]: you should create this dir and put the mib in it

File: includes/definitions/discovery/sophos-xg.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+mib: SOPHOS-XG-MIB
+modules:
+   os:
+      version:
+         - .1.3.6.1.4.1.2604.5.1.1.3.0
```
Comment (Jellyfrog) [inline includes/definitions/discovery/sophos-xg.yaml:5]: you should use the name instead, like so;
```suggestion
      version: SOPHOS-XG-MIB::sfosDeviceFWVersion
```

File: includes/definitions/sophos-xg.yaml (hunk 1 above)
Comment (corsoblaster) [reply includes/definitions/sophos-xg.yaml:13]: @murrant I didn't found how to create a folder under mibs, could you help me, or do your own and move MIB file into it?
Thanks

Comment (Jellyfrog) [reply includes/definitions/sophos-xg.yaml:13]: You create a folder under mibs and put it there

Comment (murrant) [reply includes/definitions/sophos-xg.yaml:13]: also, mib_dir is not needed, if it matches the os name. BUT, this mib should just be in the sophos folder.

Comment (corsoblaster) [reply includes/definitions/sophos-xg.yaml]: Understood.
Done.

Comment (corsoblaster) [reply includes/definitions/sophos-xg.yaml]: Directory renamed to **sophos**. 
Still present mib_dir because os name do not match directory name.

**Review by Jellyfrog — APPROVED** [review]

**Comment by murrant** [comment]
This pull request has been mentioned on **LibreNMS Community**. There might be relevant details there:

https://community.librenms.org/t/v1-69-release-changelog-october-2020/13837/1

**Pull request merged**